tools/actions_local_runner.py

```diff
@@ -164,27 +165,15 @@ async def run_mypy(files: Optional[List[str]], quiet: bool) -> bool:
     if files is not None:
         # Running quick lint, use mypy-wrapper instead so it checks that the files
         # actually should be linted
-        stdout = ""
-        stderr = ""
-        passed = True
-
-        # Pass each file to the mypy_wrapper script
-        # TODO: Fix mypy wrapper to mock mypy's args and take in N files instead
-        # of just 1 at a time
-        for f in files:
-            f = os.path.join(REPO_ROOT, f)
-            f_passed, f_stdout, f_stderr = await shell_cmd(
-                [sys.executable, "tools/mypy_wrapper.py", f]
-            )
-            if not f_passed:
-                passed = False
 
-            if f_stdout != "":
-                stdout += f_stdout + "\n"
-            if f_stderr != "":
-                stderr += f_stderr + "\n"
+        exit_code, mypy_issues = mypy_wrapper.run(args=[], files=[
+            os.path.join(REPO_ROOT, f) for f in files
+        ])
+        passed = exit_code == 0
 
-        print_results("mypy (skipped typestub generation)", passed, [stdout, stderr])
+        print_results("mypy (skipped typestub generation)", passed, [
```

Comment:
This should still call it via `shell_cmd` since that spins off a separate process to do the actual computation that we can `await`, whereas I think this will block the runner thread until mypy is done.

Reply:
Even as an async job it still all runs in the one Python thread, if Make decides to run the mypy target before any others then they would get blocked by mypy (it's the same as old multi-threading in Python under the GIL vs. `multiprocessing`), so we kind of have to have another interpreter process to handle the actual mypy compute, the runner should just be dispatching jobs and collecting their results, not doing anything intensive.

Reply:
Ok gotcha, I'll make that change (that'll probably also fix the weird `import` issue I've been hitting in the "Test tools" workflow)